internal/reversertransformer/reversertransformer_test.go

```diff
@@ -0,0 +1,107 @@
+package reversertransformer
+
+import (
+	"context"
+	"io"
+	"testing"
+
+	"github.com/apache/arrow/go/v16/arrow"
+	"github.com/apache/arrow/go/v16/arrow/array"
+	"github.com/apache/arrow/go/v16/arrow/memory"
+	pb "github.com/cloudquery/plugin-pb-go/pb/plugin/v3"
+	internalPlugin "github.com/cloudquery/plugin-sdk/v4/internal/servers/plugin/v3"
+	"github.com/cloudquery/plugin-sdk/v4/plugin"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/metadata"
+)
+
+var mem = memory.NewGoAllocator()
+
+func TestReverserTransformer(t *testing.T) {
+	p := plugin.NewPlugin("test", "development", GetNewClient())
+	p.SetSkipTableValidation(true)
```

Comment:
This is annoying 👀  a plugin doesn't know what it does, so it doesn't know that it has to automatically skip this validation if it's a transformer plugin. I don't like that whoever implements a Transformer plugin has to remember to skip this.